Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/GraphView/GraphViewConstructPresets.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <GraphCanvas/Types/ConstructPresets.h>
```

Comment:
Should the `Q_MOC_RUN` guard be placed in `ConstructPresets.h` instead to protect against misuse for other would-be users of that header?